convex/lib/slug.ts

```diff
@@ -0,0 +1,9 @@
+const ALPHABET = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
+
+export function generateSlug(length: number = 4) {
+  var acc = [];
+  for (let i = 0; i < length; i++) {
+    acc.push(ALPHABET.charAt(Math.floor(Math.random() * ALPHABET.length)));
+  }
+  return acc.join("");
+}
```

Comment:
Is a `Slug` synonymous to a hash? If so, should we be using a third-party library hash that can take another input such as `userId` which can better avoid collisions?

Reply:
a slug is basically just the code to join a room (e.g. `ABCD`). I think realistically it should be okay to keep it as is right now, since users aren't able to set their own codes (so the risk of collision isn't too high), but we could revisit this if we wanted something more bulletproof